chord_metadata_service/patients/tests/test_api.py

```diff
@@ -658,6 +659,12 @@ def setUp(self):
                     individual.age_unit = age_unit if age_unit else ""
                     individual.save()
 
+    def test_render_age_missing_time_key(self):
+        # age returns None when the time_key is missing
+        test_individual = Individual.objects.create(**c.VALID_INDIVIDUAL)
+        result = render_age(getattr(test_individual, 'time_at_last_encounter', {}), 'time_at_last_encounter')
```

Comment:
first argument should be individual, right?

Reply:
Yes! Thank you for pointing it :)